### DISCUSSION TITLE
Mouse Click Through

### DISCUSSION THREAD

**Discussion by Zenzou** [WowOpenBox/WowOpenBox#141] [Ideas]
Could window transparency and mouse click through be added?
My idea was in the stacked client(s) scenario you could make the top layer client(s) transparent and be able to click a specific window underneath.

**Comment by mooreatv** [comment]
If you have a game or app that has "holes" or transparent parts (non rectangular shape) it would already work 

But I'm not sure how one would take a game window and "make it" transparent?
